src/components/App.jsx

```diff
@@ -1,37 +1,36 @@
 import React, { Component } from 'react';
 import { Link, Route, Redirect } from 'react-router-dom';
 import { Button } from 'reactstrap';
+import Top from '../containers/Top';
 
 class App extends Component {
+  constructor(props){
```

Comment:
If you write only `super(props)` in constructor, I think you don't need to write constructor.